RiotSwiftUI/Modules/Room/Composer/View/Composer.swift

```diff
@@ -132,15 +132,15 @@ struct Composer: View {
         }
         .clipShape(rect)
         .overlay(rect.stroke(borderColor, lineWidth: 1))
-        .animation(.easeInOut(duration: resizeAnimationDuration), value: wysiwygViewModel.idealHeight)
+        // .animation(.easeInOut(duration: resizeAnimationDuration), value: wysiwygViewModel.idealHeight)
```

Comment:
we need to add a way to have different animation settings depending on context because this bit helps having smoother resize animation in non-fullscreen mode

I wonder if there is a way to get an overall better animation setup here that would work both way, otherwise just removing this depending on context seems fine

Reply:
Actually I put this back, and the animation feels a lot better, I think I commented it because I was testing for some stuff, but now it's back.